.github/workflows/linux_meson.yml

```diff
@@ -35,7 +40,14 @@ jobs:
     - name: Install dependencies
       run: |
         pip install -r build_requirements.txt
-        sudo apt-get install -y libopenblas-serial-dev
+        # Install OpenBLAS
+        set -xe
+        chmod +x tools/wheels/cibw_before_build.sh
+        if [[ $USE_NIGHTLY_OPENBLAS == "true" ]]; then
+          sudo RUNNER_OS=${{ runner.os }} tools/wheels/cibw_before_build.sh .
+        else
+          sudo RUNNER_OS=${{ runner.os }} tools/wheels/cibw_before_build.sh .
+        fi
```

Comment:
What is the difference between the two stanzas and why is the extra `chmod` now needed?

Also: this is meant to only install OpenBLAS. But sourcing `cibw_before_build.sh` does more
- installs a gfortran on macOS
- mangles the license file

Maybe you could copy the 3 needed lines (on linux) here directly, without calling a script. You would need to conditionally add a `--nightly` argument to the first line 
```
if ... 
  target=$(python tools/openblas_support.py --nightly)
else
  ...
fi
sudo cp -r $target/lib/* /usr/lib
sudo cp $target/include/* /usr/include
```

Reply:
I had forgot the missing `--nightly` flag, but yes your solution seems nicer anywho.